apps/studio/styles/fonts.css

```
@@ -22,7 +37,49 @@
   font-display: swap;
 }
 
+/* Manrope — headings (--font-heading). Latin + Latin-ext subsets, standard
+ * Google Fonts unicode-ranges. No italic axis. */
+@font-face {
+  font-family: 'Manrope';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Manrope" (font-family-name-quotes)

(font-family-name-quotes)